ArchiSteamFarm/SharedInfo.cs

```diff
@@ -69,6 +69,15 @@ public static class SharedInfo {
 	internal const string UpdateDirectory = "_old";
 	internal const string WebsiteDirectory = "www";
 
+	[PublicAPI]
+	public static readonly char[] ArrayComma = { ',' };
+
+	[PublicAPI]
+	public static readonly string[] ArrayDots = { ".." };
+
+	[PublicAPI]
+	public static readonly string[] ArrayNewlines = { "\r\n", "\r", "\n" };
```

Comment:
Can we rename these into `ListElementSeparators`, `RangeIndicators` and `NewLineIndicators`?

Reply:
I don't see a reason why not, I can consider it.